pkgs/development/python-modules/fuzzywuzzy/default.nix

```diff
@@ -0,0 +1,23 @@
+{ lib, buildPythonPackage, fetchPypi, python-Levenshtein
+, pytest, hypothesis, enum34, pycodestyle }:
+
+buildPythonPackage rec {
+  pname = "fuzzywuzzy";
+  version = "0.17.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0qid283ysgzn3pm6pdm9dy9mghsa511dl5md80fwgq80vd3xwjbg";
+  };
+
+  propagatedBuildInputs = [ python-Levenshtein ];
+
+  checkInputs = [ pytest hypothesis enum34 pycodestyle ];
```

Comment:
I'll be pushing this without `enum34` because it's only needed for the tests in python 3.3.